build_tools/circle/build_doc.sh

```diff
@@ -176,6 +176,11 @@ conda create -n $CONDA_ENV_NAME --yes --quiet \
     joblib memory_profiler packaging seaborn pillow pytest coverage
 
 source activate testenv
+# Pin PyWavelet to 1.1.1 that is the latest version that support our minumum
+# NumPy version required. If PyWavelets 1.2+ is installed, it would require
+# NumPy 1.17+ that trigger a bug with Pandas 0.25:
+# https://github.com/numpy/numpy/issues/18355#issuecomment-774610226
+pip install PyWavelets==1.1.1
```

Comment:
we need a comment here, or in our min dependency file, which would remind us to remove this once we bump our min numpy requirement.